control/veda_val.py

```diff
@@ -395,6 +402,64 @@ def send_200(self, val_api_return):
                 )
             )
 
+    def update_transcript_status(self, video_id, job_id, lang_code, name, transcript_format, provider):
+        """
+        Update status for a completed transcript.
+        """
+        if self.val_token is None:
+            self.val_tokengen()
+
+        post_data = {
+            'video_id': video_id,
+            'language': lang_code,
+            'transcript_url': name,
+            'transcript_format': transcript_format,
+            'provider': provider,
+        }
+
+        response = requests.post(
+            self.auth_dict['val_transcript_create_url'],
+            json=post_data,
+            headers=self.headers,
+            timeout=20
+        )
+
+        if not response.ok:
+            LOGGER.error(
+                'update_transcript_status failed -- job_id=%s -- video_id=%s -- provider=% -- status=%s',
+                job_id,
+                video_id,
+                provider,
+                response.status_code
+            )
+
+    def update_video_transcript_status(self, video_id, status, job_id):
```

Comment:
`update_video_status` – As we might also be using this to just update any video statuses